playbooks/site.yaml

```diff
@@ -40,3 +40,4 @@
 - import_playbook: ha_log_cfg_sync.yaml
 - import_playbook: ha_openlabcmd_install.yaml
 - import_playbook: ha_healthchecker.yaml
+- import_playbook: force-to-enable-services.yaml
```

Comment:
nit: Move to it to the line above HA related tasks. I hope HA related tasks are at  the last order.

Reply:
done